src/components/BurgerMenu/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import styles from './burger-menu.scss';
+
+const BurgerMenu = React.createClass({
+  getClassName (defaultClass) {
+    const opened = this.props.opened
+      ? styles.opened
+      : this.props.wasOpened
+      ? styles.closed
+      : '';
+
+    return `${defaultClass} ${opened}`;
+  },
+
+  render () {
+    return (
+      <div
+        className={this.getClassName(styles['burger-wrapper'])}
+        onClick={this.props.onClickHandler}
+      >
+        <span>Menu</span>
+
+        <div className={styles['burger-menu']}>
```

Comment:
Later: can we make this it's own component, with a state active or not? And style that way?
It looks awesome though!

Also. some `aria-hidden` or `presentational` attrs might be needed :)